projects/js-packages/components/components/action-popover/styles.module.scss

```diff
@@ -0,0 +1,49 @@
+.wrapper {
+	width: 304px;
+	min-height: 230px;
```

Comment:
could we define these values based on the `--spacing-base` too?

```suggestion
	width: calc( var( --spacing-base ) * 38 ); // 304px
	min-height: calc( var( --spacing-base ) * 29 ); // almost 302px!;
```

Reply:
hmm! don't think we should use `spacing` with `sizes`, these are different things when we talk about design, and if in some moment we decide to update the spacing value we shouldn't affect the sizes.

Reply:
yeah, you're right. What I'd like to guarantee is defining the sizes also based on  multiples of eight